lingua_franca/format.py

```diff
@@ -566,3 +567,53 @@ def nice_response(text, lang=''):
         assertEqual(nice_response_de("10 ^ 2"),
                          "10 hoch 2")
     """
+
+@localized_function(run_own_code_on=[FunctionNotLocalizedError])
+@localized_function()
+def nice_relative_time(when, relative_to=None, lang=None):
```

Comment:
If we're adding in a new function, we really need unit tests for it.

It would really be better as a separate PR since it goes beyond the scope of adding Basque.

Reply:
Test added and made separate PR for this (https://github.com/MycroftAI/lingua-franca/pull/231).

Now Basque changes PR only contains Basque language stuff.

The nice_relative_time PR should be merged before the Basque one, since the Basque one localizes also the nice_relative_time function and this must therefore exist.